src/views/Player.vue

```diff
@@ -74,9 +74,12 @@ var timeout = null;
 
 // wait this much time (secs) then show error page
 // if browser is not supported
-
 var browserCheckTime = 10;
 
+// how many seconds to step back (currentTime) when a user comes back to
+// a plio for a new session
+var stepBackTime = 5;
```

Comment:
Use `const` rather than `var` since it's a constant. Will use less memory. Same for `browserCheckTime` above

Reply:
Also use all caps for constants that aren't allowed to change, and define them in one place at top of file